### PR TITLE
ServiceBus : Update SDK for Swagger changes

### PR DESCRIPTION
**Pull request by v-Ajnava**
## Description
Update SDK for : 
Swagger property fixes
Namespace patch operation added 
Version update

Swagger PR Links : 
 https://github.com/Azure/azure-rest-api-specs/pull/879
https://github.com/Azure/azure-rest-api-specs/pull/1057
https://github.com/Azure/azure-rest-api-specs/pull/1138

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [X] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by shahabhijeet** [comment]
@v-Ajnava I had forgotten to delete xproj and project.json.
Please delete those files and restore .csproj file.
In VS2017 .csproj are the project files, we will not be needing project.json, global.json and *.xproj files.

**Comment by shahabhijeet** [comment]
@v-Ajnava can we get a sign off from sazeesha

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Can you generate using the commit ids for the swagger that was checked in for this change.
And after that we need version bump for the newly generated sdk

**Comment by v-Ajnava** [comment]
Done the requested changes.

File: src/SDKs/ServiceBus/Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.csproj [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+    <Version>0.2.0-preview</Version>
```
Comment (shahabhijeet) [inline src/SDKs/ServiceBus/Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.csproj]: @v-Ajnava <versionPrefix> is good enough for the version info. Either use versionPrefix or version but not both.
Also remove all commented code from .csproj files

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/ServiceBus/Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.csproj (hunk 1 above)
Comment (v-Ajnava) [inline src/SDKs/ServiceBus/Management.ServiceBus/Microsoft.Azure.Management.ServiceBus.csproj]: Used VersionPrefix and removed the commented code.

**Pull request merged**